<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWidget</class>
 <widget class="QWidget" name="SettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Настройки</string>
  </property>
  <layout class="QVBoxLayout" name="iSettingWidgetVbox">
   <item>
    <layout class="QHBoxLayout" name="iContentHbox">
     <item>
      <widget class="QListWidget" name="settingsList">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="settingsContent">
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="currentIndex">
        <number>-1</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="iControlSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="restoreBtn">
       <property name="text">
        <string>Сбросить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveBtn">
       <property name="text">
        <string>Сохранить</string>
       </property>
       <property name="icon">
        <iconset theme="document-save">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeBtn">
       <property name="text">
        <string>Закрыть</string>
       </property>
       <property name="icon">
        <iconset theme="window-close">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>settingsList</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
